layouts/partials/blog/right-nav.html

```diff
@@ -0,0 +1,17 @@
+{{ with .File }}
+    <p class="text-gray-500 text-xs">Share this post</p>
+    <ul class="p-0 list-none">
+        <li>
+            <a data-track="share-twitter"
+                    href="https://twitter.com/intent/tweet?text={{ $.Page.Title }}&tw_p=tweetbutton&url={{ $.Page.Permalink }}"
```

Comment:
Just to add a bit of breathing room between the icons:

```suggestion
                    class="mr-2"
                    href="https://twitter.com/intent/tweet?text={{ $.Page.Title }}&tw_p=tweetbutton&url={{ $.Page.Permalink }}"
```